antlr/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if(NOT DEFINED ANTLR4_JAR_LOCATION)
     message(FATAL_ERROR "ANTLR4_JAR_LOCATION must be defined")
 endif(NOT DEFINED ANTLR4_JAR_LOCATION)
 
-set(CMAKE_MODULE_PATH ${ANTLR4_MODULE_PATH})
+set(CMAKE_MODULE_PATH ${ANTLR4_MODULE_PATH_GEN} ${ANTLR4_MODULE_PATH_RUN})
```

Comment:
Is this an OS X thing? It's failing the CI builds. Maybe we need the CMake equivalent of an ifdef in order to do the right thing for each platform.

Reply:
I needed it to build on Ubuntu; I'll pull it out.

Reply:
I didn't plan for it to go out in this commit